src/runtime_src/core/include/xclbin.h

```diff
@@ -38,6 +38,8 @@
 #ifndef _XCLBIN_H_
 #define _XCLBIN_H_
 
+#define MAX_DDR_BANKS    4
```

Comment:
Need to add XoclUser driver changes in order to remove this macro def here

Reply:
Maybe so, but this PR cannot be submitted before all changes to xclbin.h have been removed.